node/core/pvf/execute-worker/build.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+fn main() {
+	// TODO?
```

Comment:
I assume this build.rs should be a copy of common/build.rs, which is unneeded.
The same goes to prepare worker. Keys are generated for cli s.

Reply:
I forgot to remove this file, good catch. I think we need `common/build.rs` to get the version for the version check (`if version != env!("SUBSTRATE_CLI_IMPL_VERSION")`). Right now it won't compile without build.rs but if it's actually unneeded that would be great. :)